includes/class-debug.php

```diff
@@ -13,26 +13,34 @@
 class Debug {
 
 	/**
-	 * Name of the debug log file.
+	 * The filesystem.
 	 *
-	 * @var string
+	 * @var Filesystem_Direct
 	 */
-	private static $log_file = 'debug-aspire-update.log';
+	private static $filesystem;
 
 	/**
-	 * The filesystem.
+	 * Generates a random file name for the log file.
 	 *
-	 * @var Filesystem_Direct
+	 * @return string The Log file name.
 	 */
-	private static $filesystem;
+	private static function generate_log_file_name() {
+		return '.debug-aspire-update-' . wp_generate_password( 12, false, false ) . '.log';
```

Comment:
```suggestion
		$stamp = time();
		$uniqid = wp_generate_password( 12, false, false );
		return sprintf('.debug-aspire-update-%s-%s.log', $stamp, $uniqid);
```

This helps find the most recent log file when they get copied out by scripts, since you can't always rely on the mtime.

Reply:
There will be only one log file at a time